### PR TITLE
Update artifact_autocloner.dm

### PR DESCRIPTION
**Pull request by Grant4455**
Added borers to the possible list of mobs created by the autocloner. 

Rare enough to keep them becoming anything more then a occasional occurrence, but at least adds a means of them being created in-game.

### CODE REVIEW

**Comment by skull132** [comment]
Retargeted to development. This wouldn't be something that goes straight into live.

File: code/modules/research/xenoarchaeology/artifact/artifact_autocloner.dm [hunk 1]
```diff
@@ -27,6 +27,7 @@
 		/mob/living/simple_animal/hostile/alien,\
 		/mob/living/simple_animal/hostile/bear,\
 		/mob/living/simple_animal/hostile/creature\
```
Comment (skull132) [inline code/modules/research/xenoarchaeology/artifact/artifact_autocloner.dm]: Missing comma here.

File: code/modules/research/xenoarchaeology/artifact/artifact_autocloner.dm [hunk 2]
```diff
@@ -27,6 +27,7 @@
...
+		/mob/living/simple_animal/borer/
```
Comment (skull132) [inline code/modules/research/xenoarchaeology/artifact/artifact_autocloner.dm]: Remove the `/` at the end.

**Comment by LordFowl** [comment]
Do non PC borers really do anything except shuffle around?

**Comment by Grant4455** [comment]
They do not, but I believe players who have 'be borer' enabled will be prompted to join as them as if they were a borer produced via the 'reproduce' option. I may be mistaken.

**Comment by printer16** [comment]
If that is the case then this is a bad idea. Having something like borers could change an extended round into chaos and not really extended.

**Comment by Grant4455** [comment]
Given the chances of this happening, and the fact that it can also spawn Xenomorphs (which can also be a round unto themselves), I don't think that's really that much of an issue.

The likelyhood of producing hostile mobs is 33%, then it picks one randomly from that list it'll churn out forever, to say nothing of the fact that autocloners are comparatively rare. This is going to be very occasional, but if occasionally xenoarch can turn it into a xenomorph round, I don't see why even more occasionally, it can turn it into a borer round.

**Comment by SonicGotNuked** [comment]
Non PC borers just shuffle around.

**Comment by printer16** [comment]
They can't turn it into a xenomorph round because they are non-playable xenomorphs. If they aren't player controlled it's fine, but otherwise this isn't a good idea at all. Rare or not, it can be extremely disruptive to a round if player controlled while xenomorphs spawning are stuck in that room as their AI doesn't try to escape like a human player would.

**Comment by Grant4455** [comment]
" Rare or not, it can be extremely disruptive to a round and can get annoying"

Generally I find things that are simply disruptive only get annoying if they're commonplace enough to lose novelty, so I'd like to illustrate the point

Lets say you have one serious anomalist every five rounds. That's... Very generous, but lets go with it. Lets say you have a one-in-ten chance of finding an autocloner. It's actually much worse then that. You find perhaps 2, maybe 3 anomalies per round, following procedure, doing paperwork, etc. Then you have a 1 in 3 chance of it spawning hostile mobs, then a 1 in 5 chance of it spawning a borer.

Or, we'd expect an autocloner that makes borers to pop up every 375~ rounds.

Forgive me if I feel that's rare enough to be a fun little gimmick, not something that'd become annoying.

**Comment by printer16** [comment]
Again, I don't really care how rare it is but personally I don't find this to be a good idea due to what I said above.

**Comment by Alberyk** [comment]
Cloners are kinda common, and they will give birth to an endless tide of borer, maybe a different, far more rare, artifact that can spawn a single borer?

**Comment by skull132** [comment]
Weee necro. @Grant4455 See what Alberyk said. You can probably make a new xenoarch find that is of some probability (< 25%) and spawns a borer or two and grabs ghosts for them. Infini-spawning borers doesn't sound like a bueno idea.

**Comment by Alberyk** [comment]
I could probably code it myself if you want.

**Comment by printer16** [comment]
This hasn't received an update in a while, and it's not something that is currently needed. Feel free to make changes and remake this or request one of us to reopen this, though.

**Pull request closed without merging**